pkg/controller/workloadspread/workloadspread_controller.go

```diff
@@ -369,14 +375,14 @@ func (r *ReconcileWorkloadSpread) getSuitableSubsetNameForPod(ws *appsv1alpha1.W
 		if err != nil {
 			return "", err
 		} else if matchedSubset == nil {
-			return "", nil
+			return FakeSubsetName, nil
```

Comment:
plz update the func comments ahead, now it will not return "" if suitable subset is not found

Reply:
done